src/templates/finops-hub/modules/dataExplorer.bicep

```diff
@@ -132,6 +132,74 @@ var ftkVersion = contains(ftkver, '-') ? split(ftkver, '-')[0] : ftkver
 var ftkBranch = contains(ftkver, '-') ? split(ftkver, '-')[1] : ''
 var dataExplorerPrivateDnsZoneName = replace('privatelink.${location}.${replace(environment().suffixes.storage, 'core', 'kusto')}', '..', '.')
 
+// Actual = Minimum(ClusterMaximumConcurrentOperations, Number of nodes in cluster * Maximum(1, Core count per node * CoreUtilizationCoefficient))
+var ingestionCapacity = {
```

Comment:
This feels excessive. Why can't we use the query approach?

Reply:
Agreed, but we can't set the concurrency value at runtime, only at the time of deployment.

Reply:
Added a var to determine the concurrency value based on the ADX SKU selected by the customer.